monai/apps/pathology/transforms/post/array.py

```diff
@@ -89,11 +97,11 @@ class GenerateWatershedMask(Transform):
     generate mask used in `watershed`. Only points at which mask == True will be labeled.
 
     Args:
-        softmax: if True, apply a softmax function to the prediction.
-        sigmoid: if True, apply a sigmoid function to the prediction.
-        threshold: if not None, threshold the float values to int number 0 or 1 with specified theashold.
-        remove_small_objects: whether need to remove some objects in the marker. Defaults to True.
-        min_size: objects smaller than this size are removed if `remove_small_objects` is True. Defaults to 10.
+        activation: the activation layer to be applied on the input probability map.
+            It can be "softmax" or "sigmoid" string, or any callable. Defaults to "softmax".
+        threshold: an optional float value to threshold to binarize probability map.
+            If not provided, defaults to 0.5 when activation is not "softmax", otherwise None.
+        min_object_size: objects smaller than this size are removed. Defaults to 10.
```

Comment:
Might want to clarify what 10 relates to - i.e. pixel area (rather than radius, width etc)